src/renderer/views/Watch/Watch.js

```diff
@@ -82,6 +82,7 @@ export default defineComponent({
       isUpcoming: false,
       isPostLiveDvr: false,
       isUnlisted: false,
+      license: null,
```

Comment:
Please reset the property back to null in the `$route` watcher.

Reply:
done 👍🏽